New package: BcubeVTK v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: BcubeVTK
- Repository: https://github.com/bcube-project/BcubeVTK.jl
- Created by: @bmxam
- Version: v0.1.0
- Commit: 943c073f8280c45be980c2819fd31e16e42ec827
- Reviewed by: @bmxam
- Reference: https://github.com/bcube-project/BcubeVTK.jl/issues/6#issuecomment-2514957431
- Description: Implementation of `Bcube` IO interface for VTK format
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598dfe0a6eff13a833e65f9b8c0368908843b9f3cbbbed4e85fdeba4f625f90802b3c2c459d230010bb32ab151b384041b4aa5323b0166dd8c85256e308aff3c891790676cc0a99f546602e37e09f7cde0f50cc4a1387fcbd17e1a95e213109925c4bac00737c79f39dbc6a3fcf9c2c6edb0b7b6be028c71383d613411b1ce9b5be -->

goerz commented:
Thank you for submitting your package! Could you add a little bit of documentation before registering? [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), in addition to the description that's already there, that would be a small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though.

bmxam commented:
Hi Michael,

I agree. I will update the README with more info and a short example. However, I won't push anything before monday (but anyway I guess the PR is blocked until I push something).

Just for your information, this is actually a "plugin" for the main package [Bcube.jl](https://github.com/bcube-project/Bcube.jl) who is instrumented with Documenter & Cie.

goerz commented:
>  I won't push anything before monday (but anyway I guess the PR is blocked until I push something.

Yes, that's right. No rush!

> Just for your information, this is actually a "plugin" for the main package [Bcube.jl](https://github.com/bcube-project/Bcube.jl) who is instrumented with Documenter & Cie.

Sometimes it's good enough to have a brief README with just a paragraph that points people to the documentation of the main project. The only point is that looking at the README, someone should have some idea of the context and if/how it can be used.

[noblock]

bmxam commented:
@goerz I've updated the README.md. Should I re-trigger the tagbot with `@JuliaRegistrator register` ?

JuliaRegistrator committed:
New package: BcubeVTK v0.1.0

UUID: f7a61acb-f482-4505-b2eb-e9cadd76b6b4
Repo: https://github.com/bcube-project/BcubeVTK.jl.git
Tree: 9c95d815c1e93ac5cfdc3589b1fefec1429201dd

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
You can, if you want to make sure that the tagged version has the updated README. Otherwise, it's optional. Do make sure to edit all your comments to contain `[noblock]`, though.

[noblock]

bmxam commented:
oh ok, thanks for the tip

[noblock]
